src/open_inwoner/scss/components/Sitemap/sitemap.scss

```diff
@@ -0,0 +1,5 @@
+.sitemap {
+  section {
+    padding-top: 1em;
```

Comment:
And yeah, we should not use static margins, but the variables instead, which you need to know, but I can clean it up.